tsconfig.json

```diff
@@ -9,5 +9,15 @@
         "resolveJsonModule": true,
         "esModuleInterop": true,
         "lib": ["esnext", "dom"]
+    },
+    "paths": {
+        "@soramitsu-ui/icons": ["packages/icons/src"],
```

Comment:
It's correct to point "pure" import to `lib.ts` file inside of `src` dir, e.g. `"@soramitsu-ui/icons": ["packages/icons/src/lib.ts"]`